Blog post about the new release (#973)
datenanfragen/website: content/en/blog/v2-release/index.md

{

>     "title": "Datenanfragen v2: Refreshed interface",

Reviewer: ```suggestion
    "title": "datarequests.org v2: Refreshed interface and complaint generator",
```

    "type": "blog",
    "date": "2022-12-06T12:00:00+02:00",
    "description": "Today we release the next version of our website with many improvements: A new face for request generator, more comprehensive request history, a fancy new frontpage and a generator for complaints and admonitions.",
    "featured_image": "",
    "tags": [ "complaint generator", "new user experience", "update", "open source" ],
    "authors": [ "zner0L" ]
}

> In an old Christian tradition, German children get presents in their shoes on the morning of the 6th of December. Today, we put something in your shoes: The new release of our websites with many great new features! We spent a lot of time in the last few months to update the user interface and took great care to make it as easy and accessible as possible to send requests to comapnies and other data controllers. Thanks to the [Prototype Fund](https://prototypefund.de/en/) run by the [Open Knowledge Foundation Germany](https://okfn.de/en/) and funded by the [German Federal Ministry of Education and Research](https://www.bmbf.de/bmbf/en/home/home_node.html), who gave a grant to Lorenz and Benni, we were able to put more time than usual into this release (read more in our {{< link slug="verein/transparency#12-external-grants-that-members-have-received" text="transparency information" >}}). Now we are just really excited and proud to present you all the new features. If you want to take a look at the code, you can [find all changes on GitHub](TODO: Link to release).

Reviewer 2: ```suggestion
In an old Christian tradition, German children get presents in their shoes on the morning of the 6th of December. Today, we put something in your shoes: The new release of our websites with many great new features! We spent a lot of time in the last few months to update the user interface and took great care to make it as easy and accessible as possible to send requests to companies and other data controllers. Thanks to the [Prototype Fund](https://prototypefund.de/en/) run by the [Open Knowledge Foundation Germany](https://okfn.de/en/) and funded by the [German Federal Ministry of Education and Research](https://www.bmbf.de/bmbf/en/home/home_node.html), who gave a grant to Lorenz and Benni, we were able to put more time than usual into this release (read more in our {{< link slug="verein/transparency#12-external-grants-that-members-have-received" text="transparency information" >}}). Now we are just really excited and proud to present you all the new features. If you want to take a look at the code, you can [find all changes on GitHub](TODO: Link to release).
```

Reviewer: comapnies -> companies

Not sure why that isn't being highlighted.

## Multi-page request generator

We completely redesigned the generator flow to make it less cluttered and also easier to send lots of requests to different companies. While we are keeping the old one-page-generator up at {{< link slug="/g" text="/g" >}} for those who prefer it, in the new interface we split the request generation process into several pages and we guide you through the process of sending your requests. In the company selection phase, we also now recommend packs of companies that collect the data of lots of people, so that new users have it easier to start requesting. In the next step, you can now also choose what sub-entities of the controller you want to include in your request and mark if you want to enquire about tracking and don't know how the controller can identify you.

## Comprehensive request history

> If you use our website regularly, you might be familiar with the “My requests” page. If you didn't disable it in the {{< link slug="/privacy-controls" text="privacy controls" >}}, we save all the requests you sent in your browser and give you an overview of them. Up until now, this was just a pretty convoluted table, but for years we wanted to improve upon it and rebuild it as a proper request management page. In our new release, we create a proceeding for each of your requests which holds all messages you sent relating to it, e.g. admonitions and complaints. You can add information about messages you received from the controller, so you have it all in one place once you want to complain. We save a state for all requests, so you know your current progress, and we remind you with a badge on all pages if there are requests overdue you should send an admonition for.  

Reviewer: ```suggestion
If you use our website regularly, you might be familiar with the “My requests” page. If you didn't disable it in the {{< link slug="/privacy-controls" text="privacy controls" >}}, we save all the requests you sent in your browser and give you an overview of them. Up until now, this was just a pretty convoluted table, but for years we wanted to improve upon it and rebuild it as a proper request management page. In our new release, we create a proceeding for each of your requests which holds all messages you sent relating to it, including admonitions and complaints. You can add information about messages you received from the controller, so you have it all in one place if you need to send a complaint. We save the state of all requests, so you know your current progress, and we remind you with a badge on all pages if there are requests overdue you should send an admonition for.  
```

## Multi-page request generator

We completely redesigned the generator flow to make it less cluttered and also easier to send lots of requests to different companies. While we are keeping the old one-page-generator up at {{< link slug="/g" text="/g" >}} for those who prefer it, in the new interface, we split the request generation process into several pages and we guide you through the process of sending your requests. In the company selection phase, we also now recommend packs of companies that collect the data of lots of people, so that new users have it easier to start requesting. In the next step, you can now also choose what sub-entities of the controller you want to include in your request and select whether you want to enquire about tracking and don't know how the controller can identify you.

> {{< img name="multi-page-generator-en" alt="Screenshot of three pages of the multi-page-generator (from left to right): the first image shows the request type choosing dialog containg the options “Get access to data about me.”, “Delete (parts of) data about me.”, “Correct data about me.” and “Stop receiving direct marketing”. The second image shows the company choosing page with a search bar on top and several suggestion packs under it, with the titles: “Social media and communication”, “Entertainment”, “Finance” and “Telecommunication”. The third image show the identification data dialog with several input fields asking for identification data for “Datenanfragen.de e. V.“ such as name, email address and address. Under the input fields there is a button to send the request." caption="Three pages of the new generator: The request type chooser, the company search and the view for filling in you identifcation data." >}}

Reviewer: ```suggestion
{{< img name="multi-page-generator-en" alt="Screenshot of three pages of the multi-page-generator (from left to right): the first image shows the request type choosing page listing the options “Get access to data about me.”, “Delete (parts of) data about me.”, “Correct data about me.” and “Stop receiving direct marketing”. The second image shows the company choosing page with a search bar on top and several suggested company packs under it, with the titles: “Social media and communication”, “Entertainment”, “Finance” and “Telecommunication”. The third image show the identification data page with several input fields asking for identification data for “Datenanfragen.de e. V.“ such as name, email address and address. Under the input fields, there is a button to send the request." caption="Three pages of the new generator: The request type chooser, the company search and the view for filling in your identification data." >}}
```

## Comprehensive request history

If you use our website regularly, you might be familiar with the “My requests” page. If you didn't disable it in the {{< link slug="/privacy-controls" text="privacy controls" >}}, we save all the requests you sent in your browser and give you an overview of them. Up until now, this was just a pretty convoluted table, but for years we wanted to improve upon it and rebuild it as a proper request management page. In our new release, we create a proceeding for each of your requests which holds all messages you sent relating to it, including admonitions and complaints. You can add information about messages you received from the controller, so you have it all in one place if you need to send a complaint. We save the state of all requests, so you know your current progress, and we remind you with a badge on all pages if there are requests overdue you should send an admonition for.  

> {{< img name="my-requests" alt="Screenshot of the “My requests”-page. Under the explanation text there are several requests with different states, such as “Action needed”, “Overdue” and “Done”. One request is folded out showing two messages, one from the user and one from the requested company." caption="The new request history on the “My requests“-page." >}}

Reviewer: ```suggestion
{{< img name="my-requests" alt="Screenshot of the “My requests” page. Under the explanation text, there are several requests with different states, such as “Action needed”, “Overdue” and “Done”. One request is expanded showing two messages, one from the user and one from the requested company." caption="The new request history on the “My requests“ page." >}}
```

## Complaint generator & request reactions

The new “My requests” page shines even more when used in combination with the feature we are most proud of: Based on the history of your request, we offer you the option to “react”. If you choose to do so, we help you assess what missteps a controller might have taken by asking you some questions and then generate an admonition or even a proper complaint from your answers. While we want to be clear that we are no lawyers and can't offer help in specific cases, we do hope this makes it easier for you to actually complain to the authorities. You are not alone with being overwhelmed by controller replies anymore.

> {{< img name="complaint-generator" alt="Screenshot of the admonition generator. On top, it shows the question “Has there been a problem with your request?” and below that are several buttons featuring kinds of problems. One example of a button text: “Company has not answered yet.”" caption="First page of the admonition generator" >}}

Reviewer: ```suggestion
{{< img name="complaint-generator" alt="Screenshot of the admonition generator. On top, it shows the question “Has there been a problem with your request?” and below that are several buttons featuring different kinds of problems. One example of a button text: “Company has not answered yet.”" caption="First page of the admonition generator" >}}
```
